lib/api/client.rb

```diff
@@ -144,7 +144,7 @@ def upload_to_connector(connector_id, filepath, run_now = true, max_retries = 3,
           query_response = RestClient::Request.execute(
             method: :post,
             url: connector_endpoint,
-            headers: headers,
```

Comment:
RestClient's version didn't bump - how do we have a different signature?

Reply:
Are these in response to a rubocop? Also, note the trailing colon - we've got `headers:,` instead of `headers,`

Reply:
Yes rubocop say that now we need write ```headers:,``` instead of ```headers: headers,```